Text/Mustache/Compile.hs

```diff
@@ -62,12 +63,32 @@ compileMustacheDir pname path =
 --
 -- @since 0.2.2
 
+compileMustacheDirCustom :: MonadIO m
```

Comment:
The `@since 0.2.2` note for `complieMustacheDirCustom` is obviously incorrect. It should be `@since 1.2.0` instead. On the other hand `getMustacheFilesInDir` should have `@since 0.2.2`.

----

Please try to implement `compileMustacheDirCustom` similar to what you have right now and then define `compileMustacheDir` in terms of it, I believe it should be just:

```
compileMustacheDir = compileMustacheDirCustom "mustache"
``` 

For this it makes sense for `compileMustacheDir` to take the extension as the first argument.

Reply:
Makes sense, will do.